Simong expedia select seats

gmsimon22 opened the pull request:
Here is my procedure on selecting flight seats on Expedia.

braydenmarsh commented:
Hi Lauren! I think it would be helpful to use a screenshot for part three because the other parts include screenshots as well. I think it would add more uniformity to the document. I think splitting it into 3 parts works great. Maybe even add names to each part if you want (for style). Otherwise, really good work.

Eden-Miller commented:
Hi Lauren!

Like I told Brayden! I want to thank you for your flexibility and understanding on this project considering the situation. I know this review won't be helpful for the initial assignment submission, but I'm hoping it will be helpful for a potential resubmission, future reference, or to help us successfully complete our current project. 

- I would consider your **warning** a staging step, not an alerting step. It may be helpful to format it differently. It seems to operate more as a prerequisite. Plus, having a warning message at the beginning of a document doesn't really conform to procedure genre conventions (but there's always a time and a place to break conventions). 
- I'm curious as to why you divided your procedure into parts. I don't think it's necessary, but it does help break up the document so you aren't hit with a text wall. Maybe consider naming them something that pertains to the steps underneath them.
- I think your coaching moves are great! I think we can edit the copy for better usability, which is something we've been working on in our topic model. 
- It would be helpful to be able to clearly identify your staging and alerting steps via formatting. 

Great work on your procedure. 

Thank you again!
Eden Miller

gmsimon22 committed:
added outline

added an outline and main headings to the select seats branch

gmsimon22 committed:
added images

added images to select seats image folder

gmsimon22 committed:
added content and images

I added the content (procedure) and included guiding screenshots

gmsimon22 committed:
changed image markdown to html

I changed the image markdown to html with image sizing

gmsimon22 committed:
small changes to content

updated some italics and a note

dmbrasw2 commented on simong/p-expedia-select-seats.md:
```diff
@@ -0,0 +1,44 @@
+
+## Prerequistes
+- You must select a flight option that allows for seat selection.
+    - The ability to select seats is dependent on the flight class (eg., economy plus, business).
```
Good info to include

WBlan97 replied:
I second that!

dmbrasw2 commented on simong/p-expedia-select-seats.md:
```diff
@@ -0,0 +1,44 @@
+
+1) Scroll to the 'seats' section and click on 'choose your seats'.
+
+<img src="/simong/images/images-seats/choose-seats.jpeg/" alt="choose your seat message" style="width: 300px">
```
The text is a bit difficult to read. Can you size the picture up at all/take a larger screenshot?

WBlan97 replied:
You may have to use the aspect ration command that's mentioned in the instructions.

dmbrasw2 commented on simong/p-expedia-select-seats.md:
```diff
@@ -0,0 +1,44 @@
+1) Scroll to the 'seats' section and click on 'choose your seats'.
+
+<img src="/simong/images/images-seats/choose-seats.jpeg/" alt="choose your seat message" style="width: 300px">
+    note: the green check mark indicates that seat choice is included for your flight. If there is a red X mark, then you will need to return to the flight selection section and change the flight class for each flight that you wish to have seat choice.
```
Suggest making this a bullet (-) or a block quote (>) to draw attention to it

WBlan97 replied:
I think you could even bold or italicize the whole statement too. I would consider omitting the part about the green check mark or maybe just leading with the red x mark statement. I think alerting moves work well when you lead with the possible error message.

alyssajohnsonunofficial replied:
Agreed, definitely draw attention to it with a bullet point or by bolding or italicizing it! I also agree with Wyatt that leading with the red x mark statement could be an effective way to draw attention to it more!

dmbrasw2 commented on simong/p-expedia-select-seats.md:
```diff
@@ -0,0 +1,44 @@
+
+2) Click on the seat of your choice.
+
+<img src="/simong/images/images-seats/select-seat.jpeg" alt="seating chart" style="width: 300px">
```
Good picture! Crisp and legible

alyssajohnsonunofficial replied:
Agreed, the images you use are great!

dmbrasw2 reviewed (commented):
good overall! I only had a couple little suggestions: 1 about your first pic, and one about making a note a block quote or bullet list

WBlan97 commented on simong/p-expedia-select-seats.md:
```diff
@@ -0,0 +1,44 @@
+- You must select a flight option that allows for seat selection.
+    - The ability to select seats is dependent on the flight class (eg., economy plus, business).
+
+- Depending on the airline, you may need to book the flight first before you are eligible to select seats.
```
Maybe could be good to include another list item that talks about different ways different airlines do this.

WBlan97 commented on simong/p-expedia-select-seats.md:
```diff
@@ -0,0 +1,44 @@
+
+Optional Step:
+- repeat the process for any additional flights.
+
```
Don't airlines often make it where some seats are paid and some are free? Maybe you could include another alerting move about that.

WBlan97 commented:
Another clear and easy follow procedure! Once again, I would maybe beef up the staging moves/motivation.

alyssajohnsonunofficial commented on simong/p-expedia-select-seats.md:
```diff
@@ -0,0 +1,44 @@
+A benefit to most Expedia flights is the option to select seats. The process to select seats through Expedia's site involves choosing and confirming seats. The visual process helps to make choosing the best seat a seamless experience.
+
+## Prerequistes
+- You must select a flight option that allows for seat selection.
```
Super strong and helpful prerequisite section here! This is definitely essential information for the user to know. I also love your introduction once again, you're really good at using language that's concise and professional but still interesting and user-friendly.

alyssajohnsonunofficial commented on simong/p-expedia-select-seats.md:
```diff
@@ -0,0 +1,44 @@
+
+<img src="/simong/images/images-seats/change-seats.jpeg" alt="choose seats message with change seats button" style="width: 300px">
+
+2) Follow through the same process of choosing and confirming your new seats.
```
Awesome consistent use of imperative verbs!

alyssajohnsonunofficial reviewed (commented):
This one's shaping up great, just like your other two!

gmsimon22 committed:
added PR template

I added the PR template for select seats procedure

gmsimon22 committed:
Added more content to PR Template

I added more content to PR Template

gmsimon22 committed:
Made a quote block and capitalized

I changed the Note to a quote block and I fixed some missing capitalization

gmsimon22 committed:
added spaces to line up the images with the text

I added some spaces to line up the images with the text

gmsimon22 committed:
Added to staging and alerting moves, images and captions, and content

I provided additional motivation and staging, added more content to the prerequisites section, improved overall formatting, changed image widths and added captions, and adding more alerting moves with visual cues

gmsimon22 committed:
Added readme.md and user scenario

I added the procedure summary for the readme.md, as well as added the user scenario to the user-scenario.md

gmsimon22 committed:
added numbers to headings

I added numbers to the 'choosing seats' and 'changing seats' headings

gmsimon22 committed:
added PDF and google doc links to user scenario

I added the PDF and google doc versions of my task analysis to my user-scenario.md

lingeringcode committed:
Feedback from Lindgren
